src/server/utils/file/file.ts

```diff
@@ -24,6 +24,6 @@ export function loadFileSync(filepath: string, postProcess: "json" | "yaml"): ob
     case "json":
       return JSON.parse(fileData);
     case "yaml":
-      return yaml.safeLoad(fileData);
+      return yaml.load(fileData) as object;
```

Comment:
Note for future self: https://github.com/nodeca/js-yaml/blob/ab31bba6b41f58390f431123ffec5031b986edf5/migrate_v3_to_v4.md#safeload-safeloadall-safedump--load-loadall-dump